### DISCUSSION TITLE
The new "location" and problems assigning a physical place for items

### DISCUSSION THREAD

**Discussion by bluikko** [netbox-community/netbox#6448] [Q&A] [answered]
I welcome the change from "rack group" to "location", it seems much useful at first glance, but have to question if the old name was more descriptive for how things stand currently.

I am aware of #6440 but it seems to affect other things - I may be wrong, it has happened plenty lately!

Currently as it is the "location" function is not very useful since things cannot be assigned to it. Example: power feeds, clusters.

Is it only an oversight that they can only be assigned to a "site", or I have understood wrong the purpose of "location"? Or this would be in scope for #6440?

For example: I have a building with multiple floors and multiple computer rooms + closets. Some of these have their own clusters, power panels + feeds, and so on. The only way they can be modeled currently is by setting each of them as "sites" and not "locations". As a workaround I will move everything back from "locations" to "sites" but it would be nice to know what is the direction planned for "location".

**Comment by ryanmerolle** [comment, marked as answer]
Raise a Feature request and explain your use case.  I just think these models are missing the locations relationship because it was not a popular use case hence no one brought it up.

**Reply by bluikko** [reply]
So you would say that it is possible I have not misunderstood how locations should be used?

**Reply by ryanmerolle** [reply]
No I said it’s not a feature currently and feel free to raise a request.
